pubsub/azure/eventhubs/metadata.yaml

```diff
@@ -96,10 +96,4 @@ metadata:
     required: true
     description: |
       Storage container name.
-    example: '"myeventhubstoragecontainer"'
-  - name: consumerId
-    type: string
-    required: true # consumerGroup is an alias for this field, let's promote this to default
-    description: |
-      The name of the Event Hubs Consumer Group to listen on.
```

Comment:
This description would be lost even though it's helpful to know that consumerID here means consumer group.